### DISCUSSION TITLE
Send reply to slash command

### DISCUSSION THREAD

**Discussion by Tumpa-Prizrak** [Rapptz/discord.py#8042] [Q&A] [answered]
Now i trying to do simle `ping` command, but the way I trying doesn't work:
```py
@bot.tree.command()
async def ping(interaction: discord.Interaction):
    await interaction.channel.send("Pong from d.py!")
```
The result is [this](https://cdn.discordapp.com/attachments/961178797770670100/975755768747921458/unknown.png)
Can you help me to fix this?

**Comment by LeoCx1000** [comment, marked as answer]
You need to respond to the interaction itself.
In this case, use `interaction.response.send_message` 
([InteractionResponse docs](https://discordpy.readthedocs.io/en/latest/interactions/api.html#discord.InteractionResponse))

**Reply by Tumpa-Prizrak** [reply]
Thank you for this answer!
